addons/mail/tests/test_mail_channel.py

```diff
@@ -194,7 +194,11 @@ def test_channel_members(self):
         self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
         self.assertEqual(channel.channel_partner_ids, self.test_partner)
 
-        channel._action_remove_members(self.test_partner)
+        self.env['mail.channel.partner'].sudo().search([
+            ('partner_id', 'in', self.test_partner.ids),
+            ('channel_id', 'in', channel.ids)
+        ]).unlink()
+        channel.invalidate_cache(fnames=['channel_partner_ids', 'channel_last_seen_partner_ids'])
```

Comment:
Thank you for the PR. Is this line still necessary? I would think not?

Reply:
@seb-odoo 
Yes you were right..
I have removed that code.
and Thanks for the suggestion..